src/app/shared/models/game.model.ts

```diff
@@ -0,0 +1,7 @@
+export interface Game {
+  firstTeam_id: number;
+  secondTeam_id: number;
+  timeLeft: number;
+  time: string;
+  winner_id: number | null;
```

Comment:
You don't need to set `null ` type since every primitive accepts nullable values.
Off docs: "By default null and undefined are subtypes of all other types. That means you can assign null and undefined to something like number."

Reply:
Not if enabled all strict type checking options.

See more: https://www.typescriptlang.org/tsconfig#strict